### PR TITLE
upgrade mediaelement

### PR DESCRIPTION
**Pull request by thrijith**
updated to 4.2.8

### CODE REVIEW

File: app/main/RTMedia.php [hunk 1]
```diff
@@ -990,14 +990,11 @@ function create_table_error_notice() {
 
 	function enqueue_scripts_styles() {
 		global $rtmedia;
```
Comment (thrijith) [inline app/main/RTMedia.php]: @sanketio  @PatelUtkarsh  please see if this is ok

File: app/main/RTMedia.php [hunk 2]
```diff
@@ -990,14 +990,11 @@ function create_table_error_notice() {
...
-		if ( wp_script_is( 'wp-mediaelement', 'registered' ) ) {
-			wp_enqueue_style( 'wp-mediaelement' );
-			wp_enqueue_script( 'wp-mediaelement' );
-		} else {
-			wp_enqueue_script( 'wp-mediaelement', RTMEDIA_URL . 'lib/media-element/mediaelement-and-player.min.js', '', RTMEDIA_VERSION );
-			wp_enqueue_style( 'wp-mediaelement', RTMEDIA_URL . 'lib/media-element/mediaelementplayer.min.css', '', RTMEDIA_VERSION );
-			wp_enqueue_script( 'wp-mediaelement-start', RTMEDIA_URL . 'lib/media-element/wp-mediaelement.js', 'wp-mediaelement', RTMEDIA_VERSION, true );
-		}
+
+		wp_deregister_script('wp-mediaelement');
```
Comment (PatelUtkarsh) [inline app/main/RTMedia.php]: Instead of deregistering can we enqueue our own script?

Comment (PatelUtkarsh) [reply app/main/RTMedia.php]: `wp-mediaelement` use the different handle.

Comment (thrijith) [reply app/main/RTMedia.php]: when i tried without  deregistering core js  gets loaded which is causing issues

Comment (PatelUtkarsh) [reply app/main/RTMedia.php]: With different handle?

**Pull request merged**